packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/default-rest/actual.js

```diff
@@ -0,0 +1,15 @@
+const a = 1;
+function rest(b = a, ...a) {
```

Comment:
Hmm, that is weird especially that firefox returns `undefined`, because either I would expect the outer `a` or the inner one but not `undefined`.